### PR TITLE
Some small edits

### PR DESCRIPTION
**Pull request by mafiya69**

### CODE REVIEW

**Comment by mafiya69** [comment]
@leosartaj have a look!

File: rollcall/main.py [hunk 1]
```diff
@@ -47,7 +50,7 @@ def get_json_file(sub):
     returns json as a dictionary
     """
     if not fileExists(sub):
-        raise exce.SubjectError("Subject: %s does not exit" %(sub))
+        raise exce.SubjectError("Subject: %s does not exists" %(sub))
```
Comment (sumitsahrawat) [inline rollcall/main.py]: Does not *exist*

**Comment by sumitsahrawat** [comment]
I would now give some meaningful commit messages.
You should also squash these commits into one, as you're using 9 commits for 15-20 lines.

**Comment by mafiya69** [comment]
I am unaware of tests at time. I will do that.

**Pull request closed without merging**